stubs/Pillow/PIL/WebPImagePlugin.pyi

```diff
@@ -1,13 +1,15 @@
 from typing import Any, ClassVar
-from typing_extensions import Literal
+from typing_extensions import Literal, TypeAlias
 
 from .ImageFile import ImageFile
 
 SUPPORTED: bool
+_XMP_Tags: TypeAlias = dict[str, _XMP_Tags]
```

Comment:
This is a non-terminating recursive definition, i.e. impossible. I think this should be:

```suggestion
_XMP_Tags: TypeAlias = dict[str, str | _XMP_Tags]
```

Reply:
You are right! Thanks.